google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/slow/ITBackupTest.java

```diff
@@ -468,11 +468,7 @@ public void test02_RetryNonIdempotentRpcsReturningLongRunningOperations() throws
         try {
           client.cancelOperation(op.getName());
         } catch (SpannerException e) {
-          // Ignore UNIMPLEMENTED errors, as it seems that Cloud Spanner cannot cancel a restore
-          // operation that has already started.
-          if (e.getErrorCode() != ErrorCode.UNIMPLEMENTED) {
-            throw e;
-          }
+          // Ignore, this is expected as it cannot find the backup that we are trying to restore.
```

Comment:
```suggestion
          // Ignore, this can happen, as the restore operation sometimes fails to start if there
          // are already too many restore operations running on the instance.
```

Reply:
Thanks @olavloite for suggesting this, it definitely makes things clearer. I've changed it slightly from my side.